recipe/yum_requirements.txt

```diff
@@ -1,6 +1,3 @@
 mesa-libGL
-mesa-libGL-devel
-mesa-libEGL-devel
 mesa-dri-drivers
```

Comment:
Why can’t all of these be removed too?

Reply:
They can and should.

Reply:
All of them should be removed.

Reply:
I also believe that we need a rerender for these changes to take effect, don't we?

Reply:
I guess the tests don't need any EGL implementation. If they did, we would need them